ansible_builder/main.py

```diff
@@ -44,8 +44,13 @@ def build_command(self):
         ]
 
     def build(self):
-        self.create()
-        return run_command(self.build_command())
+        self.containerfile.prepare_galaxy_steps()
+        self.containerfile.write()
```

Comment:
It's very possible that the code errors after this, but before the final `.write()` succeeds. In that case, the user needs to know. I would suggest that we add a comment to the end of the Containerfile that it's incomplete. It would also be good to add debug statements before each write. Both would be best.